requirements/V2_REQUIREMENTS.md

```diff
@@ -0,0 +1,69 @@
+
+The WoT Profiles specification should complement the [Thing Description](https://w3c.github.io/wot-profile/#bib-wot-thing-description11) specification, by enabling ad-hoc interoperability through the use of "profiles". A profile prescribes a finite set of extensions and defaults that a Thing can be constrained to in order to guarantee out-of-the-box interoperability with any Consumer which implements that profile. Out-of-the-box interoperability means that a [Consumer](https://w3c.github.io/wot-profile/#dfn-consumer) is guaranteed to be able to use every capability of a [Thing](https://w3c.github.io/wot-profile/#dfn-thing), without Thing-specific customization.
+
+Profiles will be designed specifically for new ("greenfield") implementations where developers have the freedom to conform to a prescriptive protocol binding and set of common constraints, in order to benefit from this additional level of interoperability.
+
+## Use Cases
+
+As a developer of a WoT Consumer, I want to guarantee out-of-the-box interoperability with a wide range of WoT Things.
+
+As a developer of a WoT Thing, I want to guarantee out-of-the-box interoperability with a wide range of WoT Consumers.
+
+### Smart Home
+
+As the developer of DIY smart home hub software, I want to safely expose an HTTP-based API for monitoring and controlling a home which conforms to a common standard, so that cloud services from multiple vendors can add value to a user's smart home without having to implement a vendor-specific API.
+
+As the developer of a smart home mobile app I want to offer my users out-of-the-box interoperability with a wide range of smart home devices so that I can maximise the audience for my app.
+
+### Smart Buildings
+
+As the developer of a smart building hub, I want to safely expose an HTTP-based API for monitoring and controlling a commercial building which conforms to a common standard, so that cloud services from multiple vendors can consume data from that hub without having to implement a vendor-specific API.
+
+As the developer of a smart building cloud service, I want to be able to provide data analytics for smart buildings using smart building hubs from different vendors which conform to a common standard, rather than having to implement multiple vendor-specific APIs.
+
+### Smart Cities
+
+### Manufacturing
+
+### Energy
+
+### Transportation
+
+### Agriculture
+
+## Requirements
+
+### WoT Profiles Specification
+
+The WoT Profiles specification:
+1. MUST define a mechanism by which the author of a WoT Thing Description can denote that the Thing it describes conforms to a particular profile
+2. MUST define a registry of profiles to which WoT Things and WoT Consumers may conform
+3. SHOULD aim to keep the number of registered profiles as small as possible by rejecting profiles that significantly overlap in technologies or use cases, in order to minimise fragmentation on the Web of Things
+### Individual Profiles
```

Comment:
These are not dependent nor subsequent requirements; each stands on its own. Therefore, this should be an unordered list (bullets) not ordered (numbers). 

If there is a specific need to refer back to individual requirements, each requirement in these two lists (including the sub-requirements) SHOULD be given its own permanent identifier (URI), which can be used in future documents here and elsewhere.
```suggestion
* MUST define a mechanism by which the author of a WoT Thing Description can denote that the Thing it describes conforms to a particular profile
* MUST define a registry of profiles to which WoT Things and WoT Consumers may conform
* SHOULD aim to keep the number of registered profiles as small as possible by rejecting profiles that significantly overlap in technologies or use cases, in order to minimise fragmentation on the Web of Things

### Individual Profiles
```

Reply:
I started the Use Cases & Requirements document as a Markdown file for simplicity, and as far as I know GitHub Markdown doesn't support anchors in list items. A numbered list seems like a reasonable way of making individual requirements referenceable.

If someone wants to turn this into a more formal HTML document using ReSpec (which supports such things) then feel free, but I've [done this before](https://w3c.github.io/web-thing-protocol/requirements/) and for this kind of document I tend to think it just makes the document harder to edit.

Reply:
> GitHub Markdown doesn't support anchors in list items. 

Drat. Markdown constantly surprises me with features it lacks, especially its site-to-site variance.

> A numbered list seems like a reasonable way of making individual requirements referenceable.

It's reasonable *if* it's reasonably-well guaranteed that the members of such lists will never change order or meaning, or at least that substitute "blanks" that say something like "this item has been deleted in favor of..." will be inserted to ensure that external references will always reach the appropriate item *or* such a "blank". Even then, it would be better to have number+short-title referenceability.

All things considered, including the [W3C Priority of Constituencies](https://www.w3.org/TR/design-principles/#priority-of-constituencies), I would strongly advise conversion to [ReSpec](https://respec.org/docs/) or [Bikeshed](https://speced.github.io/bikeshed/), with an appropriate anchor for each requirement.

Reply:
Bikeshed feels the easiest way forward IMHO.